pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -343,111 +344,71 @@ func getMutualTLSMode(mtls *v1beta1.PeerAuthentication_MutualTLS) model.MutualTL
 // replaced with config from workload-level, UNSET in workload-level config will be replaced with
```

Comment:
i didn't see these section updates. i expect there' will be some.

Reply:
Why this is need to be updated? It is just a conversion from user facing enum to our internal implementation so we can maintain the same semantic between different API version.

Reply:
actually i mean line 341.

Reply:
> if there are more than 1 policy define
// port-level mTLS for the same port, the stronger one is used.

this is no longer true. we use the entire workload policy ordered by timestamp now.

Reply:
Ah, done.